src/app/home-page/widget/organization-box/organization-box.component.scss

```diff
@@ -1,18 +1,6 @@
 @use '@angular/material' as mat;
 @import '/src/materialColorScheme.scss';
 
-img[alt='User avatar'] {
-  width: 4rem;
-  height: 4rem;
-  border-radius: 50%;
-  border: solid 1px mat.get-color-from-palette($dockstore-app-gray, 4);
-  margin: 0.8rem;
-}
```

Comment:
Moved this style as well to the shared sheet (realized the org. box doesn't display avatar images in the first place... adding that to the org. ui fixes ticket)